docs/platforms/react-native/user-feedback/index.mdx

```diff
@@ -1,9 +1,73 @@
 ---
 title: Set Up User Feedback
-description: "Learn more about collecting user feedback when an event occurs. Sentry pairs the feedback with the original event, giving you additional insight into issues."
+description: "Learn how to enable User Feedback in your app."
 sidebar_order: 6000
 ---
 
+The User Feedback feature allows you to collect user feedback from anywhere inside your application at any time, without needing an error event to occur first.
+
+Note that if you're using a self-hosted Sentry instance, you'll need to be on version 24.4.2 or higher in order to use the full functionality of the User Feedback feature. Lower versions may have limited functionality.
+
+## User Feedback Widget
+
+The user feedback widget allows users to submit feedback from anywhere inside your application.
+
+To collect user feedback from inside your application use the `showFeedbackForm` method.
+
+```javascript
+import * as Sentry from "@sentry/react-native";
+
+Sentry.showFeedbackForm();
+```
+
+<Alert>
+
+Note that the `showFeedbackForm` method depends on the React Native `Modal` implementation. It is supported fully in the legacy architecture. For the new architecture (Fabric renderer) it requires React Native `0.71` and up.
+
+</Alert>
+
+To configure the form you can use the `Sentry.feedbackIntegration({})` or add it to your Sentry initialization.
+
+```javascript
+import * as Sentry from "@sentry/react-native";
+
+Sentry.init({
+  dsn: "___PUBLIC_DSN___",
+
+  integrations: [
+    Sentry.feedbackIntegration({
+        // Additional SDK configuration goes in here, for example:
+        styles:{
+            submitButton: {
+                backgroundColor: '#6a1b9a',
+            },
+        },
+        namePlaceholder: 'Fullname',
+    }),
+  ],
+});
+```
+
+There are many options you can pass to the integration constructor. See the [configuration documentation](/platforms/react-native/user-feedback/configuration/) for more details.
+
+### Feedback Form Component
+
+You can also integrate the `FeedbackForm` component manually in your app.
+
+```javascript
+import { FeedbackForm } from "@sentry/react-native";
+
+<FeedbackForm/>
+```
+
+For the full set of options you can pass to the `FeedbackForm` component see the [configuration documentation](/platforms/react-native/user-feedback/configuration/).
+
+<Alert>
+
+Note that when the device is offline, the feedback will be stored locally and sent when the device is back online.
```

Comment:
As discussed in https://github.com/getsentry/sentry-react-native/pull/4364#pullrequestreview-2542242903